source/deployment/ssl-client-certificate.rst

```diff
@@ -0,0 +1,125 @@
+SSL Client Certificate Setup
+==================================================
+
+Follow these steps to configure SSL client certificates for your browser, Mattermost Desktop Apps on Windows, Mac and Linux, and the Mattermost Classic Apps on iOS and Android.
```

Comment:
My guess would be the Classic apps do not work.  We should mention the RN apps do not work.

Reply:
Actually maybe say its a know issue that this time and is not supported.